include/zephyr/devicetree.h

```diff
@@ -17,9 +17,21 @@
 #define DEVICETREE_H
 
 #include <devicetree_generated.h>
-
 #include <zephyr/sys/util.h>
 
+#ifndef _ASMLANGUAGE
+#include <stdint.h>
+#endif
+
+/**
+ * @brief Macro to add ULL postfix to the devicetree address constants
+ */
+#ifndef _LINKER
```

Comment:
this should be `_LINKER` only

Reply:
Right, thanks!

Reply:
The <stdint.h> is getting included with #ifndef _ASMLANGUAGE, e.g in zephyr/sys/util.h.
Otherwise we'll get the following error:

```
...
 /__w/zephyr/zephyr/lib/libc/minimal/include/stdint.h: Assembler messages:
/__w/zephyr/zephyr/lib/libc/minimal/include/stdint.h:72: Error: bad instruction `typedef signed char int8_t'
...
```
see https://github.com/zephyrproject-rtos/zephyr/actions/runs/5198990587/jobs/9375804140